addons/udes_stock/models/stock_picking_type.py

```diff
@@ -2,9 +2,43 @@
 from odoo import fields, models
 
 
+TARGET_STORAGE_FORMAT_OPTIONS = [
+    ("pallet_products", "Pallet of products"),
+    ("pallet_packages", "Pallet of packages"),
+    ("package", "Packages"),
+    ("product", "Products"),
+]
+
+
 class StockPickingType(models.Model):
     _inherit = "stock.picking.type"
 
+    u_user_scans = fields.Selection(
+        [("pallet", "Pallets"), ("package", "Packages"), ("product", "Products")],
+        string="What the User Scans",
+        help="What the user scans when asked to " "scan something from pickings of this type",
+    )
+    u_target_storage_format = fields.Selection(
+        TARGET_STORAGE_FORMAT_OPTIONS, string="Target Storage Format"
+    )
+    u_under_receive = fields.Boolean(
+        string="Under Receive",
+        default=False,
+        help="If True, allow less items than the expected quantity in a move line.",
+    )
+    u_over_receive = fields.Boolean(
+        string="Over Receive",
+        default=True,
+        help="If True, allow additional items not in the ASN, or over the expected quantity, "
+        "to be added at goods in.",
+    )
+    # TODO(ale): misleading, as this also depends on `u_user_scans` and
+    # and `u_target_storage_format`; consider using an enumeration
+    u_scan_parent_package_end = fields.Boolean(
+        string="Scan parent package at the end",
```

Comment:
Should capitalise the string: https://capitalizemytitle.com/

Reply:
![image](https://user-images.githubusercontent.com/7999677/159471064-c78b5040-48bc-4144-8f07-cc480e910b12.png)

Reply:
I didn't realize this website actually checks which letters to capitalise instead of capitalising all initial letters. Thanks for reminding, I will start using it now 😆